anvill/src/Lifters/FunctionLifter.cpp

```diff
@@ -115,6 +116,31 @@ static void MuteStateEscape(llvm::CallInst *call) {
   call->setArgOperand(remill::kStatePointerArgNum, undef_val);
 }
 
+static llvm::Function *GetAnvillSwitchFunc(llvm::Module &module,
```

Comment:
Add a comment here describing the purpose of this function and showing an example of what a call to this function looks like and how it's used.